Swope acm08722 (#5929)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/create_cluster_on_prem.adoc

The *OpenShift version* identifies the version of the {ocp-short} image that is used to create the cluster. If the version that you want to use is available, you can select the image from the list of images. If the image that you want to use is not a standard image, you can enter the URL to the image that you want to use. See _Release images_ to learn more.

> When you select {ocp-short}, an option to select *Install single-node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. 

Reviewer: I guess 4.9 is old enough.

```suggestion
Select *Install single-node OpenShift (SNO)* if you are installing a single-node OpenShift cluster. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. 
```
